lib/views/help/privacy.it.html.erb

```diff
@@ -59,33 +59,18 @@ Ma chiedilo solo se è davvero necessario: purtroppo non abbiamo la possibilità
 
 </dd>
 
-<dt id="full_address">Mi hanno chiesto l'indirizzo di casa! <a href="#full_address">#</a> </dt>
+<dt id="delete_requests">Potete cancellare la mia richiesta o nascondere il mio nome? <a href="#delete_requests">#</a> </dt>
 
 <dd>
-<p>Verificare la legge italiana</p>
 
-</dd>
-
-<dt id="postal_answer">No, no, vogliono l'indirizzo per spedirmi una risposta cartacea! <a href="#postal_answer">#</a> </dt>
-
-<dd>
-<p>verificare legge italiana</p>
-
-
-</dd>
-
-<dt id="delete_requests">Potete cancellare la mia richiesta o nascondere il mio nome?  <a href="#delete_requests">#</a> </dt>
```

Comment:
Hi Gareth,

I submitted another pull request explaining the changes.

2015-02-16 12:35 GMT+01:00 Gareth Rees <notifications@github.com>:

> In lib/views/help/privacy.it.html.erb
> <https://github.com/mysociety/dirittodisapere-theme/pull/63#discussion_r24745501>
> :
>
> >
> > -</dd>
> > -
> > -<dt id="postal_answer">No, no, vogliono l'indirizzo per spedirmi una risposta cartacea! <a href="#postal_answer">#</a> </dt>
> > -
> > -<dd>
> > -<p>verificare legge italiana</p>
> > -
> > -
> > -</dd>
> > -
> > -<dt id="delete_requests">Potete cancellare la mia richiesta o nascondere il mio nome?  <a href="#delete_requests">#</a> </dt>
>
> This section is required as it's linked to from elsewhere on the site.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mysociety/dirittodisapere-theme/pull/63/files#r24745501>
> .
>



-- 



-----------------------------------------------------------------------

Antonella Napolitano
Cell. 339/8182993
LinkedIn:  <http://www.linkedin.com/in/antonellanapolitano>
http://www.linkedin.com/in/antonellanapolitano
Blog:  <http://svaroschi.blogspot.com/>http://svaroschi.blogspot.com
Twitter:  <http://twitter.com/svaroschi>http://twitter.com/svaroschi

Make a loan on Kiva <http://www.kiva.org/invitedby/svaroschi> - A place for
piloting innovations in person-to-person lending